EDAM_dev.owl

```diff
@@ -54806,7 +54813,7 @@ ows re-sequencing of complete genomes of any given organism with high resolution
         <obsolete_since>1.3</obsolete_since>
         <oldParent rdf:resource="http://www.w3.org/2002/07/owl#Thing"/>
         <oboInOwl:consider rdf:resource="http://edamontology.org/topic_3307"/>
-        <oboInOwl:hasDefinition>Theoretical biology</oboInOwl:hasDefinition>
+        <oboInOwl:hasDefinition>Theoretical biology.</oboInOwl:hasDefinition>
```

Comment:
This kind of **"fake"** definitions should be prioritised for improving. Heuristics by length < _e.g._ 40 chars?

Reply:
- Regex search `<oboInOwl:hasDefinition>.{0,41}</oboInOwl:hasDefinition>` has unfortunately 488 matches 😥

- `<oboInOwl:hasDefinition>.{0,35}</oboInOwl:hasDefinition>` still 311 😓

- `<oboInOwl:hasDefinition>.{0,31}</oboInOwl:hasDefinition>` 209 😰

- and even `<oboInOwl:hasDefinition>.{0,23}</oboInOwl:hasDefinition>` has 62 matches 😭

Reply:
@hmenager